source/includes/sync-settings-field-disabled.rst

```diff
@@ -0,0 +1,2 @@
+You cannot modify this field while Device Sync is enabled. You must :ref:`pause
```

Comment:
What option is this referring to?

Reply:
this is used in the Cluster to Sync and Sync Type field descriptions

Reply:
ah, okay. I'll just include it as plain text instead of using the shared content. Thanks for catching!